.github/workflows/publish.yml

```diff
@@ -115,18 +115,31 @@ jobs:
           script : |
             const fe_react_version= process.env.PR_VERSION;
             const owner = 'frontegg';
-            const oauthServiceRepo = 'oauth-service'
-            const workflow_id = 'update-react-dependency.yaml'
+            const oauthServiceRepo = 'oauth-service';
+            const dashboardServiceRepo = 'dashboard';
+            const oauth_workflow_id = 'update-react-dependency.yaml';
+            const dashboard_workflow_id = 'update-frontegg-react-dependency.yaml';
 
-            const data = await github.rest.actions.createWorkflowDispatch({
-              owner,
-              repo: oauthServiceRepo,
-              workflow_id,
-              ref: 'master',
-              inputs: {
-                fe_react_version,
-              }
-            });
+            const [oauthDispatchData, dashboardDispatchData] = Promise.all[
```

Comment:
You can create an array and run on it with a map in Promise.all. I think it's better